bskyweb/templates/base.html

```diff
@@ -220,6 +220,21 @@
     .nativeDropdown-item:focus {
       outline: none;
     }
+
```

Comment:
Is it enough to change `base.html` or should changes be mirrored in `index.html`? @pfrazee

Reply:
I suspect `index.html` is for the local dev version, so I've opted to mirror it so that it works there too